packages/bot/src/services/tradeEngine/trade/Balance.js

```diff
@@ -11,8 +12,7 @@ export default Engine =>
                     balance: { balance: b, currency },
                 } = r;
 
-                this.balance = roundBalance({ currency, balance: b });
-                balanceStr = `${this.balance} ${currency}`;
+                balanceStr = getFormatedText(b, currency);
```

Comment:
there is a typo in function name, `getFormattedText`

Reply:
@reza-fs what should it be? it was `getFormatedText` aaron change it to `getFormattedText`. which one is correct?